src/js/api/order.js

```diff
@@ -402,6 +402,114 @@ ripe.Ripe.prototype.setOrderStatus = function(number, status, options, callback)
     return this._cacheURL(options.url, options, callback);
 };
 
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype.setPlayerId = function(playerId, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = this.url + "notify_info" + "?" + this._buildQuery({ player_id: playerId });

```

Comment:
Check what is being really done here, as you are not sending a `PUT` with a BODY, but rather with arguments.
- Right now: `notify_info?played_id=something`
- What you want: `notify_info` with a BODY of `{ deviceId: something }`

See similar methods, like `updateCountryGroup` or other `PUT`s.